build_deps.sh

```diff
@@ -25,39 +25,52 @@ llvm_branch=tags/RELEASE_700/final
 klee_repo=https://github.com/rsas/klee
 klee_branch=pure-bv-qf-llvm-7.0
 alive_repo=https://github.com/manasij7479/alive2.git
+z3_repo=https://github.com/Z3Prover/z3.git
+z3_branch=z3-4.8.4
```

Comment:
i guess we should change the `Dockerfile` as well to reflect that 4.8.4 is needed. Current Dockerfile builds 4.7.1 which alive2 doesn't like.

Reply:
never mind, i was looking at older version. you already fixed it